apps/src/gamelab/AnimationPicker/actions.js

```diff
@@ -12,9 +12,11 @@ var AnimationPickerAction = module.exports.AnimationPickerAction = {
  * Show the animation picker
  * @returns {{type: AnimationPickerAction}}
  */
-module.exports.showAnimationPicker = function () {
+module.exports.showAnimationPicker = function (onComplete, onCancel) {
   return {
-    type: AnimationPickerAction.SHOW_ANIMATION_PICKER
+    type: AnimationPickerAction.SHOW_ANIMATION_PICKER,
+    onComplete: onComplete,
+    onCancel: onCancel
```

Comment:
I wanted the AnimationPicker to _not_ be a connected component, in case we want to reuse it at some point outside of Gamelab.  Instead, you have to give it onComplete (success) and onCancel (error or close without picking) callbacks as props.  Here, where we use redux to wrap whether we are showing the picker or not, we put our callbacks into the store so they can be easily passed as props to the AnimationPicker.